python/pfs/drp/stella/makeFluxModelInterpolator.py

```diff
@@ -22,26 +21,6 @@ def makeFluxModelInterpolator(fluxmodeldataPath):
     nProcs : `int`
         Number of processes.
     """
-    modelSet = FluxModelSet(fluxmodeldataPath)
-
-    if "isinterpolated" in modelSet.parameters.dtype.names:
-        inputParameters = modelSet.parameters[modelSet.parameters["isinterpolated"] == 0]
-    else:
-        inputParameters = modelSet.parameters
-
-    inputParameters.sort(order=["teff", "logg", "m", "alpha"])
-
-    # make an empty array for input fluxes
-    param = inputParameters[0]
-    spectrum = modelSet.getSpectrum(param["teff"], param["logg"], param["m"], param["alpha"])
-    fluxList = np.empty(shape=(len(inputParameters), len(spectrum)), dtype=float)
-    wcs = spectrum.wavelength.toFitsHeader()
-
-    # Store fluxes of each model into a single array
-    for i, param in enumerate(inputParameters):
-        spectrum = modelSet.getSpectrum(param["teff"], param["logg"], param["m"], param["alpha"])
-        fluxList[i, :] = spectrum.flux
-
```

Comment:
There have been some significant changes to the format of the fluxmodeldata package, some coming in rapid succession. I think we need to implement some sort of versioning to ensure compatibility between the fluxmodeldata and the code, and to allow the user to immediately see the problem in the case of a discrepancy.